k8s/namespaces/juror-digital/moj-reverse-proxy/moj-reverse-proxy.yaml

```
@@ -0,0 +1,18 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: moj-reverse-proxy
+  namespace: juror-digital
+  annotations:
+    fluxcd.io/automated: "true"
```

they use automated false, https://github.com/hmcts/shared-services-flux/blob/master/k8s/prod/releases/juror-digital/moj-reverse-proxy.yaml#L8
```suggestion
    fluxcd.io/automated: "false"
```